src/components/RankingSection/RankingItem.tsx

```diff
@@ -0,0 +1,99 @@
+import styled from '@emotion/styled';
+import type { GiftItem } from '@/types/gift';
+
+interface RankingItemProps {
+  item: GiftItem;
+  rank: number;
+}
+
+const RankingItem = ({ item, rank }: RankingItemProps) => {
+  const { name, imageURL, price, brandInfo } = item;
+  return (
+    <ItemWrapper>
+      <ImageWrapper>
+        <ProductImage src={imageURL} alt={name} />
+        <RankBadge isTop3={rank <= 3}>{rank}</RankBadge>
```

Comment:
```suggestion
        <RankBadge activation={activation}>{rank}</RankBadge>
```
바깥에서 RankBadge에 대한 색상? 활성상태? 같은걸 받아오면 어떨까 싶어요 ㅎㅎ

Reply:
가령 한 페이지에 여러 개의 ranking ui가 있고 어떤 곳에서는 top5, 어떤 곳에서는 top3 처럼 노출하는 조건이 달라진다고 가정해보면 코드가 어떤식으로 변화할지 상상해보세요!